<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>contatos</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabelaCtt">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>751</width>
      <height>311</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string> </string>
     </property>
    </column>
    <column>
     <property name="text">
      <string> </string>
     </property>
    </column>
    <column>
     <property name="text">
      <string> </string>
     </property>
    </column>
    <column>
     <property name="text">
      <string> </string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnGerarPdf">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>590</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Gerar PDF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>590</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Alterar Contato</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExcluirCtt">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>590</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Excluir contato</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVoltar">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>590</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Voltar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>836</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
